### PR TITLE
nit rpg ack

### PR DESCRIPTION
**Pull request by privat**
Consider :shipit: and :+1: to also be the acknowledgment of reviewers.

### CODE REVIEW

**Comment by R4PaSs** [comment]
F%&@ yea ! :shipit: !

File: lib/github/api.nit [hunk 1]
```diff
@@ -1016,6 +1016,12 @@ abstract class Comment
 
 	# Comment body text.
 	fun body: String do return json["body"].to_s
+
```
Comment (Morriar) [inline lib/github/api.nit]: I'm really not in favor to move that in lib/

**Comment by Morriar** [comment]
That's cool but I think the is_ack method should not leak outside of nitrpg.

**Comment by privat** [comment]
@Morriar Initially I wanted to have in nitrpg only then I thought I might need it to improve my script https://github.com/privat/nit/blob/master/contrib/github_merge.nit

**Comment by Morriar** [comment]
Maybe pass the pattern as a parameter. And let the client configuration handle what kind of string is recognized as an ack?

**Comment by privat** [comment]
The point is to provide a simple,  good and sane default. If one does not like it, one could redefines it or just ignore it and look for what ine likes in the body.

**Comment by Morriar** [comment]
Alright then.

**Comment by BlackMinou** [comment]
:shipit:

**Pull request merged**